src/initializers/koa/riviere.ts

```diff
@@ -15,15 +15,18 @@ export default (config, orkaOptions: Partial<OrkaOptions>) =>
       enabled: config.riviere.enabled,
       https: true,
       level: 'info',
-      maxBodyValueChars: config.riviere.maxBodyValueChars
+      maxBodyValueChars: config.riviere.maxBodyValueChars,
+      blacklistedPaths: ['/v0.4/traces'],
```

Comment:
A regex support would be more future proof but let's iterate on a next version of reviere

Reply:
That's a good idea. I will convert it now, on both PRs